package.json

```diff
@@ -46,15 +46,21 @@
     "@turf/bbox": "^6.0.1",
     "axios": "^0.19.2",
     "classnames": "^2.2.6",
+    "esri-leaflet": "^2.3.3",
+    "esri-leaflet-geocoder": "^2.3.2",
     "formik": "^2.1.4",
     "history": "^4.10.1",
     "immutable": "^4.0.0-rc.12",
+    "leaflet": "^1.6.0",
+    "leaflet.markercluster": "^1.4.1",
     "mapbox-gl": "^1.9.0",
```

Comment:
Wird `mapbox-gl` dann noch gebraucht? Habe den css import in `src/components/Map/MapBoxGLLayer.tsx` gesehen. Vllt können wir uns ja nur die CSS ziehen und in `src/styles` verlegen? Dann haben wir direkten Zugriff auf die css und können eventuell Updates machen.

Reply:
Jo, wird noch gebraucht, der Mapbox GL Layer wird ja noch weiter verwendet